lib/AST/ProtocolConformance.cpp

```diff
@@ -348,8 +348,12 @@ bool NormalProtocolConformance::isResilient() const {
   // individual witnesses.
   if (!getDeclContext()->getSelfNominalTypeDecl()->isResilient())
     return false;
-
-  return getDeclContext()->getParentModule()->isResilient();
+  if (!getDeclContext()->getParentModule()->isResilient())
+    return false;
+  // Allows bypassing resilience checks for package
+  // decls within a package whether the loaded module
+  // was built resiliently or not.
+  return !getDeclContext()->getSelfNominalTypeDecl()->allowBypassResilienceInPackage();
```

Comment:
Isn't this unnecessary given that getSelfNominalTypeDecl()->isResilient() already checks allowBypassResilienceInPackage() as well?